docs/files.md

```diff
@@ -79,13 +79,13 @@ class BookAPI:
 
 ## Interfaces
 
-Your domain may need to communicate with another domain. That domain can be in another web server across the web, or it could be within the same server. It could even be a third-party service. When your domain needs to talk to other domains, you should define **all interactions with it in the `interfaces.py` file**. Combined with APIs (see above), this forms the bounded context of the domain, and prevents domain logic leaking in.
+Your domain may need to communicate with another domain. That domain can be in another web server across the web, or it could be within the same server. It could even be a third-party service. When your domain needs to talk to other domains, you should define **all interactions the other domain in the `interfaces.py` file**. Combined with APIs (see above), this forms the bounded context of the domain and prevents domain logic from leaking in.
```

Comment:
all interactions **to** the other domain in the `interfaces.py` file

Reply:
Thanks, I've fixed this one.